main/RadioButton/index.tsx

```diff
@@ -75,9 +75,9 @@ const getCircleStyles = (size: number, color: string): ICircleProps => {
   return {
     color,
     size,
-    innerSize: size / 2 - size / 8,
+    innerSize: Math.floor(size - size / 3),
```

Comment:
Could you give an explanation of how these changes improve this component?

Reply:
It makes that inner Circle has more large ratio than before.  It is designed by Sarah.

<img width="405" alt="Screen Shot 2020-07-09 at 11 56 29" src="https://user-images.githubusercontent.com/58724686/86996212-03495d80-c1e6-11ea-9c57-89e6564c2f5e.png">

Reply:
![image](https://user-images.githubusercontent.com/27461460/86998142-b4ea8d80-c1ea-11ea-9670-e774074c2295.png)

Thanks for the detail. However, it still looks not identical to the new design.

Reply:
Color of the circle, disabled styled and the font size are bit different.

Reply:
Also, the changes on the design weren't stated in the `PR`. I think this can be enhanced in the future work.